pages/activity/[name]/manage/evaluation.tsx

```diff
@@ -0,0 +1,165 @@
+import { observer } from 'mobx-react';
+import { NewData } from 'mobx-restful';
+import { InferGetServerSidePropsType } from 'next';
+import { createRef, FormEvent, PureComponent } from 'react';
+import {
+  Badge,
+  Button,
+  Card,
+  Col,
+  Form,
+  InputGroup,
+  ListGroup,
+  Row,
+} from 'react-bootstrap';
+import { formToJSON } from 'web-utility';
+
+import { ActivityManageFrame } from '../../../../components/Activity/ActivityManageFrame';
+import { TeamAwardList } from '../../../../components/Team/TeamAwardList';
+import activityStore from '../../../../models/Activity';
+import { Award, AwardAssignment } from '../../../../models/Award';
+import { Team } from '../../../../models/Team';
+import { withRoute } from '../../../api/core';
+
+interface EvaluationPageProps {
+  activity: string;
+  path: string;
+  awardList: Award[];
+}
+
+export const getServerSideProps = withRoute<{ name: string }>();
+
+@observer
+class EvaluationPage extends PureComponent<
+  InferGetServerSidePropsType<typeof getServerSideProps>
+> {
+  store = activityStore.teamOf(this.props.route.params!.name);
+  awardStore = activityStore.awardOf(this.props.route.params!.name);
+  form = createRef<HTMLFormElement>();
+
+  componentDidMount() {
+    this.awardStore.getAll();
+  }
+
+  handleAssign = () => {};
+  handleReset = () => {
+    const { store } = this,
+      form = this.form.current;
+    form?.reset();
+    store.clearCurrent();
+  };
```

Comment:
```suggestion
  handleReset = () => this.store.clearCurrent();
```
`<form />` 的 `reset` 事件已经清空了数据，无需再 `.reset()`